lib/usd/hdMaya/adapters/proxyAdapter.h

```diff
@@ -90,6 +90,7 @@ class HdMayaProxyAdapter
     }
 
     const SdfPath& GetUsdDelegateID() const { return _usdDelegate->GetDelegateID(); }
+    HdMayaProxyUsdImagingDelegate* GetUsdImagingDelegate() { return _usdDelegate.get(); }
```

Comment:
I would have commented that a const version might have been useful as well, but I see that HdMayaProxyUsdImagingDelegate appears to not be const-correct :(